lambdas/indexer/app.py

```diff
@@ -310,3 +316,13 @@ def consolidate(self, others: List['DocumentTally']) -> 'DocumentTally':
         assert all(self.entity == other.entity for other in others)
         return replace(self, num_contributions=sum((other.num_contributions for other in others),
                                                    self.num_contributions))
+
+
+@app.schedule(f'rate({(config.indexer_lambda_timeout + 60) // 60} minutes)')
```

Comment:
Both `rate(5 minutes 30 seconds)` and `rate(330 seconds)` caused failures with `make terraform`:

```
Error: Updating CloudWatch Event Rule failed: ValidationException: Parameter ScheduleExpression is not valid.
  on indexer/chalice.tf.json line 297, in resource.aws_cloudwatch_event_rule.retrieve_fail_messages-event:
```

Reply:
I don't like the non-linear margin this produces. For example, with indexer_lambda_timeout set to 299, this gives 5 minutes, just 1s of margin. Got anything better? Can you find any documentation that would explain why 330 seconds is considered invalid?